elasticdl/python/worker/allreduce_trainer.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+
+import tensorflow as tf
+from tensorflow.python.framework.errors_impl import UnknownError
+
+from elasticdl.python.common.constants import HOROVOD_CONFIG
+
+try:
+    import horovod.tensorflow as hvd
+except ImportError:
+    pass
```

Comment:
Why pass here?

Reply:
I want to catch the `ImportError` when we train the model using ParameterServerStrategy without Horovod.